tests/engines/registries/integration/discover/discover-test.ts

```diff
@@ -153,7 +153,7 @@ module('Registries | Integration | discover', hooks => {
     });
 
     test('query parameters', async function(this: TestContext, assert: Assert) {
-        assert.expect(3 + (QueryParamTestCases.length * 6));
+        assert.expect(1 + (QueryParamTestCases.length * 2));
```

Comment:
This test keeps failing in the CI build. It seems that `assert.expect()` doesn't count `sinon.assert()`, therefore changing the numbers fixes the tests.
I honestly have no idea why it worked before. Tried various combinations. `sinon.assert()` is working because when I change the expected called params the test fails. Also this problem only happens when you use `ember-exam` to run the tests in partitions.